recipes/pweave/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set version = "0.25" %}
+{% set sha256 = "1c0f6921196646243eb7ff9eee742305909be2bc7a5eeeb06a7d1f66cc9758c7" %}
+
+package:
+  name: pweave
+  version: {{ version }}
+
+source:
+  fn: Pweave-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/p/pweave/Pweave-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+    - Pweave = pweave.scripts:weave
+    - pweave = pweave.scripts:weave
+    - Ptangle = pweave.scripts:tangle
+    - ptangle = pweave.scripts:tangle
+    - pypublish = pweave.scripts:publish
+    - pweave-convert = pweave.scripts:convert
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - markdown
+    - pygments
+    - beautifulsoup4
+
+test:
+  imports:
+    - pweave
+    - pweave.formatters
+    - pweave.themes
+    - pweave.formatters
+    - pweave.processors
+
+  commands:
+    - Pweave --help
+    - Ptangle
```

Comment:
So we prefer to try and have `--help` or `--version` added on. Mainly because not having the option often means we get an error code.